src/components/connections.vue

```diff
@@ -30,8 +30,9 @@ export default {
   .box
     h1
       span Connections
-      button.round.add +
-    form
+      button.round.add(v-if="shouldConnectionSectionBeVisible" type="button" @click="showOrHideConnectionSection") -
+      button.round.add(v-if="!shouldConnectionSectionBeVisible" type="button" @click="showOrHideConnectionSection") +
```

Comment:
we don't need two different DOM elements here — it's only the icon changing, and we can change it with a ternary inside.